mat/lq.go

```diff
@@ -87,18 +87,23 @@ func (lq *LQ) Cond() float64 {
 // and upper triangular matrices.
 
 // LTo extracts the m×n lower trapezoidal matrix from a LQ decomposition.
-// If dst is nil, a new matrix is allocated. The resulting L matrix is returned.
-// LTo will panic if the receiver does not contain a factorization.
-func (lq *LQ) LTo(dst *Dense) *Dense {
+//
+// If dst is empty, LTo will resize dst to be r×c. If dst is
+// non-empty, LTo will panic if dst is not r×c. LTo will also panic
```

Comment:
"When dst is non-empty, LTo will panic if dst is not r×c."

Reply:
Done.